### PR TITLE
Thimble issue1875

### PR DESCRIPTION
**Pull request by sanariaz154**
Add extra padding before the start of each line in the editor

### CODE REVIEW

**Comment by humphd** [comment]
@sanariaz154 you can get rid of that `src/config.json` change by doing:

```
git checkout -- src/config.json
```

I have another patch that fixes this, but it hasn't landed yet.  This is an annoying part of the Brackets build system that we don't need.

File: src/styles/bramble_overrides.css~ [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/* Menu item goes to next line due to float:right issue on Firefox */
```
Comment (Pomax) [inline src/styles/bramble_overrides.css~]: Is this a backup file? If so, this should probably be removed from the PR (see @humphd's comment on how to do that for the config file already)

**Comment by sanariaz154** [comment]
@humphd @Pomax I kind of messed up thing from this branch so I have sent pull request from new branch here [Thimble issue1875](https://github.com/mozilla/brackets/pull/649) kindly ignore my clumsiness.

**Comment by humphd** [comment]
@sanariaz154 no need to worry, we're also happy to help if you get into any issues with git.  Glad you're working with us!

**Comment by sanariaz154** [comment]
@humphd  I would like to work more I've searched from thimble.mozilla.org' [issues](https://github.com/mozilla/thimble.mozilla.org/issues?page=1&q=is%3Aopen+is%3Aissue+label%3A%22good+first+bug%22) but most of them are assigned. Can you please suggest me some other good first issues which are not yet assigned to anyone?

**Pull request closed without merging**